infoschema/pg_catalog_test.go

```diff
@@ -0,0 +1,39 @@
+package infoschema_test
```

Comment:
Please add the correct license information

Reply:
already added